microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import re
+from pathlib import PurePath
+from typing import Any, List, Optional, Type
+
+from lisa.executable import Tool
+from lisa.operating_system import CBLMariner
+from lisa.tools import Docker, Echo, Git, Modprobe
+
+
+class CloudHypervisorTests(Tool):
+    TIME_OUT = 1800
+
+    repo = "https://github.com/cloud-hypervisor/cloud-hypervisor.git"
+
+    cmd_path: PurePath
+    repo_root: PurePath
+
+    @property
+    def command(self) -> str:
+        return str(self.cmd_path)
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    @property
+    def dependencies(self) -> List[Type[Tool]]:
+        return [Git, Docker]
+
+    def run_tests(self, test_type: str, skip: Optional[List[str]] = None) -> List[str]:
+        self.node.tools[Modprobe].load("openvswitch")
```

Comment:
if this module is not loaded by default, we'd better to unload it after testing, like in after_case

Reply:
Done.